src/components/Watchlist/__test__/container.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import WatchListContainer from '../container';
+import { getWatchListRequest } from '../../../store/watchList/actions';
+
+describe('WatchListContainer ', () => {
+  const store = configureStore()({
+    watchlist: {
+      error: false,
+      loading: true,
+      total_results: 1,
+      ids: [1],
+      current_page: 2,
+    },
+    database: {
+      movies: { 1: {} },
+    },
+  });
+
+  store.dispatch = jest.fn();
+
+  const wrapper = shallow(<WatchListContainer store={store} />);
+  const container = wrapper.dive().dive();
+  const instance = container.instance();
+
+  it('Should match its snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  it('Check call lifeCycleMethod componentDidMount', () => {
+    mount(<WatchListContainer store={store} />);
+    expect(store.dispatch).toHaveBeenCalledWith(getWatchListRequest());
+  });
+
+  it('should call getWatchListRequest action', () => {
+    instance.componentDidMount();
+    expect(store.dispatch).toHaveBeenCalledWith(getWatchListRequest());
+  });
+
+  it('Check is calling container method.', () => {
+    const wrapper = mount(<WatchListContainer store={store} />);
+    const container = wrapper.find('WatchListContainer').instance();
+    jest.spyOn(container, 'goToNextPage');
+    container.goToNextPage();
```

Comment:
This test doing nothing - you need to check if calling `goToNextPage` calls another method